media/css/privacy/firefox-privacy.scss

```diff
@@ -0,0 +1,282 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import
+    '../pebbles/includes/lib',
+    '../pebbles/includes/fonts/open-sans',
+    '../pebbles/includes/mixins/typography';
+
+* {
+    @include open-sans;
+}
+
+@font-face {
+    font-family: 'Open Sans';
+    font-weight: 600;
+    font-style: normal;
+    src: url('/media/fonts/opensans-semibold.woff2') format('woff2'),
```

Comment:
Hmmm so, in the invision doc it specifically asked for semibold but, I felt a little uncomfortable as it is not part of the default stack. I then saw some other pages sometimes include additional fonts so, thought it might be a thing we do from time to time.

With what Craig said yesterday though, I am kinda thinking twice about this now, especially seeing that you brought it up. Perhaps it is worth dropping that extra weight and request, and just using bold.

Reply:
It is up to you, but my bet is most folks won't even notice 😉

Reply:
Yup, I removed it and went with plain old bold.